### ISSUE TITLE
gold-cc-cvc-input accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/gold-cc-cvc-input#10]
Contrast: Please make the CVC text a bit darker so the contrast is improved. 

Also, I'm not able to trigger the custom error message. When this does show, is it an announcement so the user can hear what the error is with spoken feedback? 

Audit results:
[Severe] Controls and media elements should have labels (4)
[Warning] Images should have an alt attribute (4)
[Warning] Text elements should have a reasonable contrast ratio (1)
Not applicable tests (7)
Passing tests (5)

**Comment by lpalmaro** [comment]
One more question - on Win / Firefox with NVDA, the hint text for the text entry fields tells the user that the fields have auto complete. Is that intentional? this is not spoken on any other screen readers.

**Comment by notwaldorf** [comment]
Done! :tada:
